src/settings.c

```diff
@@ -20,6 +20,9 @@
 #include "pwquality.h"
 #include "pwqprivate.h"
 
+static const char pwquality_base_cfgfile[] = PWQUALITY_BASE_CFGFILE;
+static char pwquality_default_cfgfile[PATH_MAX] = PWQUALITY_DEFAULT_CFGFILE;
+
```

Comment:
These values should be inside the pwq struct.

Overriding the config file name should happen inside both base and default config paths.

The use case is for having separate configuration for different applications and they should be able to have a separate configuration in /usr/lib.